server/server/data_loader/glossary.py

```diff
@@ -0,0 +1,11 @@
+import json
+from pathlib import Path
+
+
+def load_glossary(db, additional_info_dir: Path):
+    print('Loading glossary')
+
+    glossary_collection = db['glossary']
+    with open(f'{additional_info_dir}/gloss.json') as f:
```

Comment:
you could also do `with (additional_info_dir / 'gloss.json').open() as f`. Again just as a tidbit